roles/aws/tasks/main.yml

```diff
@@ -13,30 +13,50 @@
     count_tag: '{{ launch_tags }}'
     exact_count: '{{ ec2_count }}'
     wait: true
-  tags: "aws_setup"
+  tags: 'aws-start'
 
 - name: AWS | Retrieve instances info
-  delegate_to: "localhost"
+  delegate_to: localhost
   ec2_remote_facts:
     region: eu-central-1
     filters:
-      instance-state-name: running
-      "tag:Group": '{{ ec2_group }}'
+      'tag:Group': '{{ ec2_group }}'
   register: ec2
 
 - name: AWS | Build group
-  delegate_to: "localhost"
+  delegate_to: localhost
   add_host:
     hostname: '{{ item.public_dns_name }}'
     groupname: '{{ ec2_group }}'
     private_ip: '{{ item.private_ip_address }}'
   with_items: '{{ ec2.instances }}'
 
+- name: AWS | Start instances
+  delegate_to: localhost
+  ec2:
+    instance_ids: '{{ item.id }}'
+    region: '{{ ec2_region }}'
+    state: running
+    wait: true
+  with_items: '{{ ec2.instances }}'
+  tags: 'aws-start'
```

Comment:
Lets document start and stop way of using playbook. Someone will have to do it and we want to have example with --tags part of it.